duckduckgo/BrowserServicesKit
# sync device connected names

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/0/1204469803916196/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1727
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1205
What kind of version bump will this require?: Minor

**Optional**:

Tech Design URL:
CC:

**Description**:
Returns devices returned by the login URL. Update iOS and macOS to show the new devices that are connected.

**Steps to test this PR**:
1. On iOS turn off Sync
1. On macOS turn on Sync
2. Use iOS to connect the Sync on your macOS
3. Both devices should show the other device when they connect
4. Confirm this works the opposite way around too



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
